core/templates/pages/exploration-player-page/services/audio-translation-manager.service.ts

```diff
@@ -32,16 +32,16 @@ export interface AudioTranslations {
 })
 export class AudioTranslationManagerService {
   // Audio translations for the main content of a card at the top.
-  _contentAudioTranslations: AudioTranslations = null;
-  _contentHtmlForAutogeneratedAudio: string = null;
+  _contentAudioTranslations: AudioTranslations|null = null;
```

Comment:
```suggestion
  _contentAudioTranslations: AudioTranslations | null = null;
```

Please fix here and elsewhere below.

Reply:
Okay! Changing them right away!